subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -473,6 +473,12 @@ void ll_rx_dequeue(void)
 
 	mem_release(link, &mem_link_rx.free);
 
+	/* GCC is not able to recognize comments in clause under ifdef.
```

Comment:
This could be fixed by moving the fallthrough comment out of the ifdef.

If not then I think those comments should be removed. Because the are obvious, and they don't work.